pkg/reconcilermanager/controllers/reposync_controller_test.go

```diff
@@ -75,6 +75,7 @@ const (
 var filesystemPollingPeriod time.Duration
 var hydrationPollingPeriod time.Duration
 var nsReconcilerName = core.NsReconcilerName(reposyncNs, reposyncName)
+var reconcilerDeploymentReplicaCount int32 = 1
```

Comment:
What's this for? I don't see it used.

Reply:
It is used in reposync_controller_test(this test), it was originally defined in the deployment condition unit test that I just deleted